plugin/nix-exprs/sbtix.nix

```diff
@@ -110,23 +115,28 @@ in rec {
             linkArtifact = outputPath: urlAttrs:
                 let
                   artifact =
-                    (if builtins.substring 0 5 urlAttrs.url == "file:" then
+                    if urlAttrs.type or null == "built" then
                       if localBuildsRepo == "" then
                         abort "'sbtixBuildInputs' parameter missing from 'buildSbtProgram'/'buildSbtLibrary', but local dependencies found."
```

Comment:
I think it'd be hard to generate `type = "built"` by accident, and giving this information in the error message may lead someone to think that the value is wrong instead of following the instruction to add the parameter.
If the value is to blame, it's not too hard to find it in the file, so let's keep it simple.